[2024-03-28T17:46:05.005+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cryptocurrency_data_pipeline.notify scheduled__2024-01-11T00:00:00+00:00 [queued]>
[2024-03-28T17:46:25.105+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cryptocurrency_data_pipeline.notify scheduled__2024-01-11T00:00:00+00:00 [queued]>
[2024-03-28T17:46:27.196+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-03-28T17:46:44.014+0000] {local_task_job_runner.py:121} ERROR - Received SIGTERM. Terminating subprocesses
[2024-03-28T17:46:49.266+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 143
[2024-03-29T08:21:34.091+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cryptocurrency_data_pipeline.notify scheduled__2024-01-11T00:00:00+00:00 [queued]>
[2024-03-29T08:21:34.363+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cryptocurrency_data_pipeline.notify scheduled__2024-01-11T00:00:00+00:00 [queued]>
[2024-03-29T08:21:34.383+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-03-29T08:21:34.905+0000] {taskinstance.py:2192} INFO - Executing <Task(EmailOperator): notify> on 2024-01-11 00:00:00+00:00
[2024-03-29T08:21:35.066+0000] {standard_task_runner.py:60} INFO - Started process 304 to run task
[2024-03-29T08:21:35.323+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'cryptocurrency_data_pipeline', 'notify', 'scheduled__2024-01-11T00:00:00+00:00', '--job-id', '106', '--raw', '--subdir', 'DAGS_FOLDER/cryptocurrency_data_pipeline.py', '--cfg-path', '/tmp/tmpuwpkjhi2']
[2024-03-29T08:21:35.522+0000] {standard_task_runner.py:88} INFO - Job 106: Subtask notify
[2024-03-29T08:21:36.015+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-29T08:21:36.915+0000] {task_command.py:423} INFO - Running <TaskInstance: cryptocurrency_data_pipeline.notify scheduled__2024-01-11T00:00:00+00:00 [running]> on host 75e340009066
[2024-03-29T08:21:38.441+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='dom@dataengineercafe.io' AIRFLOW_CTX_DAG_OWNER='domwatcharin' AIRFLOW_CTX_DAG_ID='cryptocurrency_data_pipeline' AIRFLOW_CTX_TASK_ID='notify' AIRFLOW_CTX_EXECUTION_DATE='2024-01-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-11T00:00:00+00:00'
[2024-03-29T08:21:39.133+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/email.py:154 RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
[2024-03-29T08:21:39.258+0000] {configuration.py:1046} WARNING - section/key [smtp/smtp_user] not found in config
[2024-03-29T08:21:39.296+0000] {email.py:270} INFO - Email alerting: attempt 1
[2024-03-29T08:21:39.680+0000] {email.py:281} INFO - Sent an alert email to ['dom@dataengineercafe.io']
[2024-03-29T08:21:40.435+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=cryptocurrency_data_pipeline, task_id=notify, execution_date=20240111T000000, start_date=20240329T082134, end_date=20240329T082140
[2024-03-29T08:21:41.110+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-29T08:21:41.886+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
